content/en/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

```diff
@@ -253,5 +253,5 @@ $ kubectl delete destinationrule egressgateway-for-wikipedia
 * Uninstall Istio from your cluster:
 
     {{< text bash >}}
-    $ istioctl x uninstall --purge
+    $ echo y | istioctl uninstall --purge
```

Comment:
This should also use -y
```suggestion
    $ istioctl uninstall --purge -y
```

Reply:
Actually, thinking about it, why do any of the calls (in the test.sh files) use `echo y`, instead of `-y`? The only reason to use echo is when the command is in an md file but you don't want the doc to use -y, but rather let the user do it manually. Only in those cases does it make sense to use `echo y | snip_xxx` so that the test won't hang waiting for input.